docs/source/conf.py

```diff
@@ -20,19 +20,29 @@
 # -- Project information -----------------------------------------------------
 
 project = "tsdat"
-copyright = "2021, Carina Lansing, Maxwell Levin"
+copyright = "2021, Battelle Memorial Institute"
 author = "Carina Lansing, Maxwell Levin"
 
 # The full version, including alpha/beta/rc tags
-release = "0.2.2"
+release = "0.2.10"
 
 
 # -- General configuration ---------------------------------------------------
 
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
-extensions = ["sphinx_rtd_theme"]
+extensions = ["sphinx_rtd_theme",
```

Comment:
What do all these extra extensions do? Why is the autoapi extension disabled? –– I'm okay with it, but it's just not clear to me why this was done.

Reply:
Most of them have to do with reading and formatting docstrings for creating an API, aka the code source links in the "Configuring Tsdat" section. I took out the autoapi cause I've never found it very helpful for understanding code, but can put it back in.

Reply:
Ah, gotcha. Just a heads-up, I don't think those extensions are working correctly on readthedocs: https://tsdat--38.org.readthedocs.build/en/38/config/hook_functions.html

If you want get these working, or remove them, that's fine with me. But I would like the autoapi extension back in so we can have an API section in the sidebar – I find that useful.

Reply:
^ Just saw your commit adding autoapi back in